modules/icingaweb2/collector.go

```diff
@@ -113,11 +116,17 @@ func Collect(c *collection.Collection) {
 		}
 	}
 
-	timestamp := "7 days ago"
+	if c.Detailed {
+		for _, file := range detailedFiles {
+			c.AddFilesIfFound(ModuleName, file)
+		}
+
+		timestamp := "7 days ago"
 
-	for name, element := range journalctlLogs {
-		if service, err := collection.FindServices(element.Service); err == nil && len(service) > 0 {
-			c.AddCommandOutput(filepath.Join(ModuleName, name), "journalctl", "-u", element.Service, "--since", timestamp)
+		for name, element := range journalctlLogs {
+			if service, err := collection.FindServices(element.Service); err == nil && len(service) > 0 {
+				c.AddCommandOutput(filepath.Join(ModuleName, name), "journalctl", "-u", element.Service, "--since", timestamp)
```

Comment:
Why not hardcoded like above?

Reply:
That's one thing I also stumbled across while implementing the detailed collection.
My idea was to create a separate function c.AddJournalLog(). 

However, using the variable `timestamp` would be the nicer way than the hardcoded interval for me. In the long run, it makes it easier to customise the interval (possibly even as an argument to give the user full control).

What do you think about this?

Reply:
In the future it might be interesting to actually have a parameter to adjust this.  But for now let's have at least the same line of code instead of three different versions.

Right now we have:

` "--since", timestamp)`
` "--since \"7 days ago\"")`
` "--since, "7 days ago, ")`

I'd use `timestamp` as a global variable, that way it can easily be converted into a flag later.

Reply:
Done with 8b74214.